sample/Archiving/public/js/host.js

```diff
@@ -1,16 +1,27 @@
-var session = OT.initSession(sessionId),
+var session = OT.initSession(apiKey, sessionId),
     publisher = OT.initPublisher('publisher'),
     archiveID = null;
 
-session.connect(apiKey, token, function(err, info) {
-  if(err) {
-    alert(err.message || err);
+session.connect(token, function(error, info) {
+  if (error) {
+    console.error('Failed to connect', error);
```

Comment:
Should we use `console.error` or `alert`(which I used from the last PR)? If later, I will adjust later